catalog/spatial/stanag4559/spatial-stanag4559-source/src/test/java/org/codice/ddf/spatial/ogc/stanag4559/catalog/source/impl/LibraryImpl.java

```diff
@@ -0,0 +1,142 @@
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.spatial.ogc.stanag4559.catalog.source.impl;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.GIAS.AccessCriteria;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.GIAS.CatalogMgrHelper;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.GIAS.DataModelMgrHelper;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.GIAS.LibraryDescription;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.GIAS.LibraryManager;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.GIAS.LibraryManagerHelper;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.GIAS.LibraryPOA;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.GIAS.OrderMgrHelper;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.GIAS.ProductMgrHelper;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.UCO.InvalidInputParameter;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.UCO.ProcessingFault;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.UCO.SystemFault;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.UCO.exception_details;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.source.impl.managers.CatalogMgrImpl;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.source.impl.managers.DataModelMgrImpl;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.source.impl.managers.OrderMgrImpl;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.source.impl.managers.ProductMgrImpl;
+import org.omg.CORBA.NO_IMPLEMENT;
+import org.omg.PortableServer.POA;
+
+public class LibraryImpl extends LibraryPOA {
+
+    private LibraryDescription libraryDescription = new LibraryDescription("localhost",
+            "US|AFRL/IFEC",
+            "NSILI|1.0");
+
+    private List<String> manager = Arrays.asList("OrderMgr",
+            "CatalogMgr",
+            "ProductMgr",
+            "DataModelMgr"
+            /* Optional :
+            "QueryOrderMgr",
+            "StandingQueryMgr",
+            "CreationMgr",
+            "UpdateMgr" */);
+
+    private POA poa_;
+
+    public LibraryImpl(POA poa_) {
+        this.poa_ = poa_;
+    }
+
+    @Override
+    public String[] get_manager_types() throws ProcessingFault, SystemFault {
+        return (String[]) manager.toArray();
+    }
+
+    @Override
+    public LibraryManager get_manager(String manager_type, AccessCriteria access_criteria)
+            throws ProcessingFault, InvalidInputParameter, SystemFault {
+
+        org.omg.CORBA.Object obj;
+
+        switch (manager_type) {
+        case "CatalogMgr":
+            CatalogMgrImpl catalogMgr = new CatalogMgrImpl(poa_);
+            try {
+                poa_.activate_object_with_id(manager_type.getBytes(), catalogMgr);
+            } catch (Exception e) {
+                // Ignore
```

Comment:
So two things, do not catch "Exception" and do not just ignore it. Log something out at an appropriate level. Also, I'd like to see some debug logging for large methods like this in all of these classes.